examples/web-wagmi/src/inbox/components/ConversationComposer.tsx

```diff
@@ -0,0 +1,51 @@
+import { Profile, ProfileOwnedByMe, useStartLensConversation } from '@lens-protocol/react-web';
+
+import { never } from '../../utils';
+
+type ConversationComposerProps = {
+  ownedProfile: ProfileOwnedByMe;
+  peerProfile: Profile;
+};
+
+export function ConversationComposer({ ownedProfile, peerProfile }: ConversationComposerProps) {
+  const { startConversation, isLoading, error } = useStartLensConversation({
+    ownedProfile,
+    peerProfile,
+  });
+
+  const submit = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    const form = event.currentTarget;
+
+    const formData = new FormData(form);
+    const content = (formData.get('content') as string | null) ?? never();
+
+    await startConversation(peerProfile.ownedBy, content);
+
+    if (!error) {
```

Comment:
curious about the flow here and where does the `error` change the value. 
If i get correctly it's technically not possible for `error` to be non nullable here as even when `startConversation` will update the `error` it would happen at the next render cycle and this function would already finish the execution?

Reply:
here is xmtp implementation, error is set when calling the startConversation function https://github.com/xmtp/xmtp-web/blob/main/packages/react-sdk/src/hooks/useStartConversation.ts

but yes, you are right, the handling of error in submit here doesn't make sense

Reply:
i see, but there is just no chance that `error` gonna be set in the same task queue so probably can be removed